wntr/epanet/toolkit.py

```diff
@@ -538,21 +513,6 @@ def ENgetflowunits(self):
         self._error()
         return iCode.value
 
-    def ENgetnodeid(self, iIndex):
-        """
-        desc: Gets the ID name of a node given its index.
-
-        :param a node's index (starting from 1).
-        :return the node's ID name.
-        """
-        fValue = ctypes.create_string_buffer(SizeLimits.EN_MAX_ID.value)
-        if self._project is not None:
-            self.errcode = self.ENlib.EN_getnodeid(self._project, iIndex, byref(fValue))
-        else:
-            self.errcode = self.ENlib.ENgetnodeid(iIndex, byref(fValue))
-        self._error()
-        return str(fValue.value, 'UTF-8')
-
```

Comment:
why was this removed?

Reply:
I don't know how this got removed, and I'm trying to fix that.